### ISSUE TITLE
Multiple FieldValue lines bug

### ISSUE THREAD

**Issue by maxguru** [mikehaertl/php-pdftk#118]
Apparently, version 0.4.2 introduced a bug.

Code to reproduce:
```
$pdf = new \mikehaertl\pdftk\Pdf($filepath);
$data = $pdf->getDataFields();
print_r($data);
```

`$filepath` contains path to [multiselect2.pdf](https://github.com/mikehaertl/php-pdftk/files/1886002/multiselect2.pdf).

Output with version 0.4.1:
```
mikehaertl\pdftk\DataFields Object
(
    [_string:mikehaertl\pdftk\DataFields:private] => ---
FieldType: Choice
FieldName: List Box
FieldFlags: 2097152
FieldValue: 1
FieldValue: 2
FieldValue: 3
FieldValue: 4
FieldJustification: Left
    [_array:mikehaertl\pdftk\DataFields:private] => Array
        (
            [0] => Array
                (
                    [FieldType] => Choice
                    [FieldName] => List Box
                    [FieldFlags] => 2097152
                    [FieldValue] => Array
                        (
                            [0] => 1
                            [1] => 2
                            [2] => 3
                            [3] => 4
                        )

                    [FieldJustification] => Left
                )

        )

    [storage:ArrayObject:private] => Array
        (
            [0] => Array
                (
                    [FieldType] => Choice
                    [FieldName] => List Box
                    [FieldFlags] => 2097152
                    [FieldValue] => Array
                        (
                            [0] => 1
                            [1] => 2
                            [2] => 3
                            [3] => 4
                        )

                    [FieldJustification] => Left
                )

        )

)
```

Output with version 0.4.2,
```
mikehaertl\pdftk\DataFields Object
(
    [_string:mikehaertl\pdftk\DataFields:private] => ---
FieldType: Choice
FieldName: List Box
FieldFlags: 2097152
FieldValue: 1
FieldValue: 2
FieldValue: 3
FieldValue: 4
FieldJustification: Left
    [_array:mikehaertl\pdftk\DataFields:private] => Array
        (
            [0] => Array
                (
                    [FieldType] => Choice
                    [FieldName] => List Box
                    [FieldFlags] => 2097152
                    [FieldValue] => 1
FieldValue: 2
FieldValue: 3
FieldValue: 4
                    [FieldJustification] => Left
                )

        )

    [storage:ArrayObject:private] => Array
        (
            [0] => Array
                (
                    [FieldType] => Choice
                    [FieldName] => List Box
                    [FieldFlags] => 2097152
                    [FieldValue] => 1
FieldValue: 2
FieldValue: 3
FieldValue: 4
                    [FieldJustification] => Left
                )

        )

)
```

An array of numbers is the correct output.  Version 0.4.2 is returning a string with multiple lines and raw pdftk output.

**Comment by mikehaertl** [comment]
I've pushed a fix to master. Can you help testing please?

If you want to do it manually, it's only a single line that has to be changed:

https://github.com/mikehaertl/php-pdftk/commit/3722f6ad794224b50a58c6586d5f15c80032f4d0#diff-b45c760ff7fe27aacafea506d21a96b8R164

**Comment by mikehaertl** [comment]
@maxguru Did you already find some time to test the fix? I'm just waiting for you confirmation to prepare a new release.

**Comment by maxguru** [comment]
I'm sorry for not responding sooner.  I can confirm the one line change (preg_match) fixed the issue for me.  Everything else seems to be working fine as far as I can tell.  Thanks.

**Comment by mikehaertl** [comment]
Great. Released 0.4.4 containing this fix. Thanks for your help!
